pom.xml

```diff
@@ -54,7 +54,7 @@
         <kotlin.version>1.9.10</kotlin.version>
         <kotlin.coroutines.version>1.7.3</kotlin.coroutines.version>
         <neo4j.version>4.4.25</neo4j.version>
-        <kafka.version>2.6.3</kafka.version>
+        <kafka.version>3.7.0</kafka.version>
```

Comment:
I think we need to stick with latest 3.0.x as that's still supported within Confluent ecosystem (https://docs.confluent.io/platform/current/installation/versions-interoperability.html). Is that version also affected with this CVE?